packages/connect-common/files/coins.json

```diff
@@ -2228,6 +2228,26 @@
     "erc20": [],
     "eth": [],
     "misc": [
+        {
+            "blockchain_link": {
+                "type": "solana",
+                "url": [
+                    "wiser-green-owl.solana-mainnet.discover.quiknode.pro/c218d0c9e451f6c5c0bf8b7c7d8e5a72384c9570/"
+                ]
+            },
+            "curve": "ed25519",
+            "decimals": 9,
+            "is_testnet": false,
+            "name": "Solana",
```

Comment:
I am aware this is not the right way to add a new coin to this json, we plan to resolve this in a separate PR which I suppose will also require an update in the `common` submodule. Actually we wanted to discuss how to approach this, maybe we can keep it like this until the final version is actually in your master? What do you think?

Reply:
sure, no problem to keep it here temporarily. but don't forget to add a PR to https://github.com/trezor/trezor-firmware